### DISCUSSION TITLE
Does pygimli support complete electrode model (CEM) or simulation of finite electrodes/contact resistance?

### DISCUSSION THREAD

**Discussion by jinshanmu** [gimli-org/pyGIMLi#661] [Q&A]
I discovered the article `Rücker, C., & Günther, T. (2011). The simulation of finite ERT electrodes using the complete electrode model. Geophysics, 76(4), F227-F238.` and I think using CEM would make the simulation more realistic. Thank you!

**Comment by halbmy** [comment]
In case of extended electrodes, CEM can more accurately describe the fields. For long electrodes, SEM is often the easier solution (Ronczka et al. 2015). It is only about mesh generation.

pyGIMLi supports CEM and SEM, but lacks examples as this goes beyond standard ERT. You might find material in the BERT repository (https://gitlab.com/resistivity-net/bert), under examples/modelling/CEM or SEM.

Ronczka, M., Rücker, C. & Günther, T. (2015): Numerical study of long electrode electric resistivity tomography – Accuracy, sensitivity and resolution. Geophysics 80(6), E317-328, doi:10.1190/geo2014-0551.1.

**Reply by jinshanmu** [reply]
Thank you for the reply! I'll look into the BERT repository.
